cclib/io/ccio.py

```diff
@@ -12,6 +12,7 @@
 
 import atexit
 import io
+import ntpath
```

Comment:
Using `os.path` will remove the need for some of `path_leaf`. It's even recommended by `ntpath`:
```
In [1]: import ntpath

In [2]: ntpath?
Type:        module
String form: <module 'ntpath' from '/home/eric/virtualenvs/cclib/lib/python3.6/ntpath.py'>
File:        ~/virtualenvs/cclib/lib/python3.6/ntpath.py
Docstring:  
Common pathname manipulations, WindowsNT/95 version.

Instead of importing this module directly, import os and refer to this
module as os.path.
```